spec/models/cinema_hall_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CinemaHall, type: :model do
```

Comment:
Delete such empty files, please.

Reply:
Well, rspec creates it by default, gonna use models test.

Reply:
I know. The final decision it's up to you. I personally keep only files that introduce any value to the project.